rpm/archivematica/archivematica.spec

```diff
@@ -126,9 +126,8 @@ Archivematica dashboard with Nginx + gunicorn.
 #
 
 %prep
-rm -rf /usr/share/archivematica
+rm -rf /usr/share/python/archivematica*
```

Comment:
Can you change the wildcard and use the package names?

Reply:
How would that look like?

    rm -rf /usr/share/python/archivematica-{dashboard,mcp-server,mcp-client,storage-service}

Would that cover all the cases?

Reply:
Ok I've changed it let me know if that works for you.